make/conf/module-loader-map.conf

```diff
@@ -94,48 +94,26 @@ PLATFORM_MODULES_windows= \
 
 NATIVE_ACCESS_MODULES= \
     java.base \
-    java.datatransfer \
     java.desktop \
     java.instrument \
-    java.logging \
     java.management \
-    java.management.rmi \
-    java.naming \
-    java.net.http \
     java.prefs \
     java.rmi \
-    java.scripting \
-    java.se \
     java.security.jgss \
-    java.security.sasl \
     java.smartcardio \
-    java.sql \
-    java.sql.rowset \
-    java.transaction.xa \
-    java.xml \
-    java.xml.crypto \
     jdk.accessibility \
-    jdk.charsets \
+    jdk.attach \
```

Comment:
The list of allowed modules has been rewritten from scratch, by looking at the set of modules containing at least one `native` method declaration.

Reply:
Should I understand this list to be the set of modules exempt from needing to specific that native access is allowed ?
ie they always have native access without any warnings, and further that any attempt to enable warnings, or to disable native access for these modules is ignored ?

Reply:
Yes, this was added via JDK-8327218. The changes in this PR are just trimming down the list to only the modules that have native code.